### PR TITLE
Deploy new versions to pypi

### PR DESCRIPTION
**Pull request by chriselion**
Deploy to pypi when new releases are made. Follows the instructions from https://circleci.com/blog/continuously-deploying-python-packages-to-pypi-with-circleci/ but with a few modifications to handle 3 packages, and make it easier to switch to test.pypi.org

### CODE REVIEW

File: gym-unity/setup.py [hunk 1]
```diff
@@ -1,15 +1,40 @@
...
 
+import os
+import sys
 from setuptools import setup, find_packages
+from setuptools.command.install import install
+
+VERSION = "0.10.1"
+
+
+class VerifyVersionCommand(install):
```
Comment (chriselion) [inline gym-unity/setup.py]: I don't have a good way to share this across the libraries :/

Comment (harperj) [reply gym-unity/setup.py]: :'(

Comment (harperj) [reply gym-unity/setup.py]: New package `ml-agents-setup-common` :troll:

Comment (chriselion) [reply gym-unity/setup.py]: It crossed my mind.

**Pull request merged**